ESP32LapTimer/src/ESP32LapTimer.cpp

```diff
@@ -74,7 +69,17 @@ void setup() {
   //PowerDownAll(); // Powers down all RX5808's
   delay(250);
 
+#if defined(WIFI_MODE_AP)
   InitWifiAP();
+#elif defined(WIFI_MODE_CLIENT)
+  if(!InitWifiClient()) {
+    Serial.println("Failed to connect to WiFi Network");
+    Serial.println("Starting up in AP mode instead!");
```

Comment:
I know we have quite some of those in the code, but I'd prefer if we switch to the logging functions (log_d, log_e etc).
This way it can be disabled (i.e. if you are using USB for communication and the program is confused with these messages)

Reply:
Resonable, enough ill change that :D

Reply:
Fixed in edfd49b